src/tests/simple_tests/testdata/uma/reads_and_writes/entry.asm

```diff
@@ -0,0 +1,82 @@
+        .text
+        .file	"Test_26"
+        .rodata.cst32
+        .p2align	5
+    CPI0_0:
+        ; this is 'user' contract
+	    .cell 800000
+        .text
+        .globl	__entry
+    __entry:
+    .main:        
+        add 65, r0, r1
+        add 120, r0, r2
+
+        ; write '120' in slot 65
+        uma.heap_write r1, r2, r0, r0
+
+        ; write '150' in slot 65 for AUX
+        add 150, r0, r2
+        uma.aux_heap_write r1, r2, r0, r0
+
+
+        ; now reading.
+        uma.heap_read r1, r0, r3, r0
+
+        ; assert r3 == 120
+        sub.s! 120, r3, r0
+        jump.ne @.panic_wrong_read
+
+
+        uma.aux_heap_read r1, r0, r3, r0
+        sub.s! 150, r3, r4
+        jump.ne @.panic_wrong_read
+
+        ; static writes & reads are not supported from kernel contracts yet.
```

Comment:
Maybe we should move this part to separate explicitly ignored test?

Reply:
good idea - done.